### ISSUE TITLE
Exception in thread "main" com.intel.analytics.bigdl.nn.abstractnn.InvalidLayer

### ISSUE THREAD

**Issue by land1725** [intel-analytics/analytics-zoo#981]
Hello,when I use Dense Layer,some errors occur ,could you help me?

Engine.init

    val hidden_1 = Dense(128, activation = "relu", inputShape = Shape(inputNumber))
    val hidden_2 = Dense(128, activation = "relu")
    val hidden_3 = Dense(1, activation = "sigmoid")

    val model = Sequential()
      .add(hidden_1)
      .add(hidden_2)
      .add(hidden_3)

    val criterion = BCECriterion()
    val estimator = new DLClassifier(model, criterion, Array(inputNumber))
      .setFeaturesCol("features")
      .setLabelCol("label")
      .setBatchSize(1280)
      .setLearningRate(0.01)
      .setOptimMethod(new Adam[Float]())
      .setValidation(Trigger.everyEpoch, testInput, Array(new AUC()), 3096)
    val dlModel = estimator.fit(trainInput.coalesce(4))
    val testRank = dlModel.transform(testInput.coalesce(4))
    testRank.show(100, false)

18/12/03 20:52:44 INFO utils.Engine$: Find existing spark context. Checking the spark conf...
Exception in thread "main" com.intel.analytics.bigdl.nn.abstractnn.InvalidLayer: Do not mix Sequential[be57a7d7]{
  [input ->  -> output]

}(isKerasStyle=false) with Layer
                           (isKerasStyle=true):
         Dense[e5d3c02e]
	at com.intel.analytics.bigdl.nn.abstractnn.InferShape$class.excludeInvalidLayers(InferShape.scala:98)
	at com.intel.analytics.bigdl.nn.abstractnn.AbstractModule.excludeInvalidLayers(AbstractModule.scala:58)
	at com.intel.analytics.bigdl.nn.abstractnn.InferShape$class.validateInput(InferShape.scala:108)
	at com.intel.analytics.bigdl.nn.abstractnn.AbstractModule.validateInput(AbstractModule.scala:58)
	at com.intel.analytics.bigdl.nn.DynamicContainer.add(DynamicContainer.scala:44)
	at HuoyingBigDL$.main(HuoyingBigDL.scala:686)
	at HuoyingBigDL.main(HuoyingBigDL.scala)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.apache.spark.deploy.JavaMainApplication.start(SparkApplication.scala:52)
	at org.apache.spark.deploy.SparkSubmit$.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:894)
	at org.apache.spark.deploy.SparkSubmit$.doRunMain$1(SparkSubmit.scala:198)
	at org.apache.spark.deploy.SparkSubmit$.submit(SparkSubmit.scala:228)
	at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:137)
	at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
18/12/03 20:52:44 INFO spark.SparkContext: Invoking stop() from shutdown hook

**Comment by land1725** [comment]
import com.intel.analytics.zoo.pipeline.api.keras.models.Sequential
import com.intel.analytics.zoo.pipeline.api.keras.layers._
the problem has been solved
